examples/python/multiprocessing/server.py

```diff
@@ -48,7 +50,11 @@ def is_prime(n):
 
 class PrimeChecker(prime_pb2_grpc.PrimeCheckerServicer):
     def check(self, request, context):
-        _LOGGER.info("Determining primality of %s", request.candidate)
+        _LOGGER.info(
```

Comment:
Do you see this message? I don't, unless I setup logger in `_run_server`. I've moved it to a separate method and called it from both places.

```py
def setup_logger():
    handler = logging.StreamHandler(sys.stdout)
    formatter = logging.Formatter("[PID %(process)d] %(message)s")
    handler.setFormatter(formatter)
    _LOGGER.addHandler(handler)
    _LOGGER.setLevel(logging.INFO)
```